components/layout_2020/dom_traversal.rs

```diff
@@ -89,6 +89,7 @@ where
     }
 }
 
+#[derive(Debug)]
```

Comment:
I've also added `Debug` support to some classes which should help to debug these issues in the future.